client/Dockerfile

```
@@ -1,12 +1,12 @@
 FROM node:24
 WORKDIR /app
-COPY package.json ./
+COPY package*.json ./
 COPY tsconfig.json ./
 # Update npm to the latest version
 RUN npm config set cache /app/.npm-cache --global
-RUN npm install -g npm@11.1.0
+RUN npm install -g npm@11.12.1
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)